### ISSUE TITLE
Error when hitting "Run all" first thing after starting the server

### ISSUE THREAD

**Issue by eliasdorneles** [Lothiraldan/balto#6]
I get an error if I try to run all tests before hitting the "Collect all", right after starting the server.

Steps to reproduce:

- start server
- start React UI, and open the browser
- click "Run all" right away (don't click Collect all)

Alternative sequence to reproduce:

- start server
- start React UI, and open the browser
- click "Collect all"
- click "Run all"
- restart the server
- refresh browser
- click "Run all"

This is the error that I see in the console:

```
ERROR:root:__init__() got an unexpected keyword argument 'run_id'
Traceback (most recent call last):
  File "/home/elias/.virtualenvs/balto37/lib/python3.7/site-packages/aiohttp_json_rpc/rpc.py", line 313, in _handle_rpc_msg
    msg=msg,
  File "/home/elias/.virtualenvs/balto37/lib/python3.7/site-packages/aiohttp_json_rpc/rpc.py", line 146, in __call__
    return await self.method(**method_params)
  File "/home/elias/src/balto/balto/server.py", line 95, in run_all
    await asyncio.gather(*tasks, loop=loop)
  File "/home/elias/src/balto/balto/suite.py", line 76, in launch_all
    await session.run()
  File "/home/elias/src/balto/balto/runners/subprocess_runner.py", line 58, in run
    await self.launch_cmd(final_cmd)
  File "/home/elias/src/balto/balto/runners/subprocess_runner.py", line 71, in launch_cmd
    _read_stream(process.stderr, self.read_line),
  File "/home/elias/src/balto/balto/runners/subprocess_runner.py", line 16, in _read_stream
    await cb(line)
  File "/home/elias/src/balto/balto/runners/base.py", line 89, in read_line
    await self.event_emitter.emit(data)
  File "/home/elias/src/balto/balto/event_emitter.py", line 19, in emit
    await asyncio.gather(*awaitables)
  File "/home/elias/src/balto/balto/server.py", line 32, in process_notification
    process_test_result(message, SUITES)
  File "/home/elias/src/balto/balto/server.py", line 53, in process_test_result
    suites[suite].update_test(message)
  File "/home/elias/src/balto/balto/store.py", line 90, in update_test
    self.tests[test_dict["id"]] = SingleTest(**test_dict)
TypeError: __init__() got an unexpected keyword argument 'run_id'
```

**Comment by Lothiraldan** [comment]
Indeed I reproduced the issue. I will commit a fix soon.

**Comment by Lothiraldan** [comment]
It has been fixed by https://github.com/Lothiraldan/balto/commit/a4cbf1e73f546484a276761b763b86cb0fcbd183 and included in the `v0.1.2` release
